### PR TITLE
align description of audit_rules_kernel_module_loading

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- add the part of the line restricting UIDs for RHEL* and OL* products
- add OCIL macro for finit_module

#### Rationale:

- aligns rule description with check and remediations
- completes OCIL

#### Review Hints:

- compile e.g. rhel9 and sle15 product
- compare their remediations, check and rule description, differences should be the same

### CODE REVIEW

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading/rule.yml [hunk 1]
```diff
@@ -8,7 +8,11 @@ description: |-
     To capture kernel module loading and unloading events, use following lines, setting ARCH to
     either b32 for 32-bit system, or having two lines for both b32 and b64 in case your system is 64-bit:
     <pre>
+        {{% if 'ol' in product or 'rhel' in product %}}
```
Comment (Mab879) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading/rule.yml:11]: Please double check the alignment on this line.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Just one minor issue.

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading/rule.yml (hunk 1 above)
Comment (vojtapolasek) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading/rule.yml]: Good catch, it added some unnecessary blank lines. I fixed it.

**Review by Mab879 — APPROVED** [review]
lgtm.

Thanks!

**Pull request merged**